libcni/api.go

```diff
@@ -284,13 +284,14 @@ func (c *CNIConfig) checkNetwork(ctx context.Context, name, cniVersion string, n
 
 // CheckNetworkList executes a sequence of plugins with the CHECK command
 func (c *CNIConfig) CheckNetworkList(ctx context.Context, list *NetworkConfigList, rt *RuntimeConf) error {
-	// CHECK was added in CNI spec version 0.4.0 and higher
-	if gtet, err := version.GreaterThanOrEqualTo(list.CNIVersion, "0.4.0"); err != nil {
-		return err
-	} else if !gtet {
-		return fmt.Errorf("configuration version %q does not support the CHECK command", list.CNIVersion)
+	if len(list.CNIVersion) > 0 {
```

Comment:
This means that if `list.CNIVersion` is empty, we will treat it as greater than 0.4.0 ?

Reply:
No, it means do version checking only if  `list.CNIVersion` is not empty.